apis/server/container_bridge.go

```diff
@@ -134,6 +134,7 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
 			ID:              c.ID,
 			Names:           []string{c.Name},
 			Image:           c.Config.Image,
+			ImageID:         c.Image,
```

Comment:
should we also change the logic in `removeImage` when compare the image

Reply:
I check the logic and found that we should use imageID to do the compare when we remove the image. There are some cases during removing image:

* rmi shortID
* rmi imageName which share same id with other one

Therefore, we use imageID to do compare to check container is using the image or not.